src/style/fonts.css

```diff
@@ -1,30 +1,36 @@
-@font-face{
-  font-family: "Sage UI";
-  src: url('https://fonts.sage.com/Sage_UI-Regular.woff2') format("woff2"), url('https://fonts.sage.com/Sage_UI-Regular.woff') format("woff");
-  font-weight: 400;
+@font-face {
+  font-family: "CarbonIcons";
+  src: url("./assets/carbon-icons-webfont.woff") format("woff");
+  font-weight: normal;
   font-style: normal;
-  font-display: swap;
 }
 
-@font-face{
+@font-face {
   font-family: "Sage UI";
-  src: url('https://fonts.sage.com/Sage_UI-Medium.woff2') format("woff2"), url('https://fonts.sage.com/Sage_UI-Medium.woff') format("woff");
-  font-weight: 700;
   font-style: normal;
-  font-display: swap;
+  font-weight: 400;
```

Comment:
The css file in the design-tokens package [here](https://github.com/Sage/design-tokens/blob/master/assets/fonts/sageui.css) uses font weights of 400, 500 and 700.

Why are we using different font weights to that package?

Reply:
Thanks for taking a look @samtjo. This is a misalignment between `design-tokens` and Carbon/legacy SOP apps, which @harpalsingh has flagged with DS and other product teams. Unfortunately if we were to use the 400/500/700 weights instead then our components won't look quite right. Therefore the plan is include this weight override for now in Carbon, then once we've updated the whole codebase to use typography tokens, we should be good to remove this override